client/assets/js/board.js

```diff
@@ -47,33 +41,36 @@ export default class Board {
 		this.apples.push(apple);
 	}
 
-	getAvailableColor(){
-		let snakeColor = this.snakes.map(function(snake){
+	getAvailableColor() {
+		let snakeColor = this.snakes.map(function (snake) {
 			return snake.color;
 		});
 
-		let availableColor =  this.color.find(function(c) {
-			if(!snakeColor.includes(c)){
+		return this.color.find(function (c) {
+			if (!snakeColor.includes(c)) {
 				return c;
 			}
 		});
-		return availableColor;
 	}
 
 	render(board) {
-		setInterval(function(){
+		setInterval(function () {
+            // TEMP: ONLY START MOVING THE FIRST SNAKE FOR TEST PURPOSES
 			board.snakes[0].move();
+            // END TEMP
+
+			board.checkSnakeWithAppleCollision();
```

Comment:
Maybe it'd be better to check for collisions inside the call to `.move`? Not sure. but it feels like moving and checking for collisions should be bundled together